packages/terra-date-input/src/DateInput.jsx

```diff
@@ -116,6 +118,8 @@ const defaultProps = {
   value: undefined,
   useExternalFormatMask: false,
   isA11yControlled: false,
+  errorMessageId: null,
+  ariadescribedby: null,
```

Comment:
Not able to find `ariadescribedby` prop in DateInput, where is it being used ?

Reply:
This is removed as per latest changes.